AppCenterIdentity/AppCenterIdentity/Internals/MSIdentityPrivate.h

```diff
@@ -12,6 +14,12 @@ static NSString *const kMSIdentityETagKey = @"MSIdentityETagKey";
 // The key for Identity auth token stored in keychain.
 static NSString *const kMSIdentityAuthTokenKey = @"MSIdentityAuthToken";
 
+// The key for the MSALAccount homeAccountId stored in user defaults.
+static NSString *const kMSIdentityMSALAccountHomeAccountKey = @"MSIdentityMSALAccountHomeAccount";
+
+// The key for the MSALAccount environment stored in user defaults.
+static NSString *const kMSIdentityMSALAccountEnvironmentKey = @"MSIdentityMSALAccountEnvironment";
```

Comment:
What is this one used for? Looks like we can get rid of it.

Reply:
good catch.